app/src/main/res/layout/activity_end_solo.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/floral_white"
+    tools:context=".activity.GameEndingActivity">
+
+
```

Comment:
It seems that there are a lot of common elements in this layout and the one of activity_end_multiple, is it possible to regroup them?

Reply:
I will leave them separate for now. I am not sure what will be displayed in the game for multiple players, so that is why they look similar.